randovania/gui/ui_files/ui_data_editor.py

```diff
@@ -0,0 +1,261 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file is generated, shouldn't have been committed.

It's also been generated in the wrong path??

Reply:
Yikes! Sorry! Maybe was due to misconfiguration at the beginning. I should have thought again why this generated file contained changes and was under version control